cpp/src/backends/protocol/otf_message.cc

```diff
@@ -1,30 +1,41 @@
-#include <arpa/inet.h>
-#include <sys/socket.h>
-#include <glog/logging.h>
-
 #include "src/backends/protocol/otf_message.hh"
 
 namespace torchserve {
-  byte_buffer OTFMessage::CreateLoadModelResponse(StatusCode code, const std::string& message) {
-    LoadModelResponse response = {
-      code, 
-      static_cast<int>(message.length()), 
-      message
-    };
-    std::byte msg[sizeof(LoadModelResponse)];
-    std::memcpy(msg, &response, sizeof(LoadModelResponse));
-    byte_buffer response_byte_buffer;
-    std::copy(response_byte_buffer.begin(), response_byte_buffer.end(), msg);
-    return response_byte_buffer;
+  void OTFMessage::EncodeLoadModelResponse(std::unique_ptr<torchserve::LoadModelResponse> response, char* data) {
+    char* p = data;
+    // Serialize response code
+    int32_t s_code = htonl(response->code);
```

Comment:
nit: do you think we can move adding integer to data pointer to a util function and re-use later below?

Reply:
Done